src/azure-cli/azure/cli/command_modules/security/custom.py

```diff
@@ -305,3 +306,21 @@ def create_security_workspace_setting(client, resource_name, target_workspace):
 def delete_security_workspace_setting(client, resource_name):
 
     client.delete(resource_name)
+
+
+# --------------------------------------------------------------------------------------------
+# Security ATP
+# --------------------------------------------------------------------------------------------
+
+
+def get_atp_setting(client, resource_group_name, storage_account_name):
+
+    return client.get('subscriptions/{}/resourceGroups/{}/providers/Microsoft.Storage/storageAccounts/{}'
```

Comment:
Use f-strings if they are supported, i.e. `client.get(f'subscriptions/{client.config.subscription_id}...)`

Reply:
f-string is introduced in Python 3.6 by [PEP 498](https://www.python.org/dev/peps/pep-0498/). Using it will break compatibility with Python 2.

Reply:
Thanks, was not aware that the CLI supports Python 2.